src/movies.js

```diff
@@ -1,16 +1,77 @@
 // Iteration 1: All directors? - Get the array of all directors.
-// _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
+function getAllDirectors (movies) {
+    return movies.map(movie => movie.director);
+    }
```

Comment:
Sweet! My only thought here is the indentation, it looks a bit funny. If you haven't done it yet, could you install Prettier in your vscode. It should help out with indentations.